### ISSUE TITLE
undefined method `name_from_mixlib' for #<LicenseAcceptance::Acceptor

### ISSUE THREAD

**Issue by s-bernard** [test-kitchen/test-kitchen#1548]
## Description

When run by latest test-kitchen (2.2.2), my tests fail with:
```
Failed to complete #converge action: [undefined method `name_from_mixlib'
for #<LicenseAcceptance::Acceptor:0x00007f002929a470>]
```
The undefined call is there:
`test-kitchen-2.2.2/lib/kitchen/provisioner/chef_base.rb:274:in check_license`

I checked in license_acceptance gem, effectively no `name_from_mixlib` can be found but instead there is `id_from_mixlib(mixlib_name)`. When I switched to it, it seems to work as expected.

## Kitchen Version

```ruby
Test Kitchen version 2.2.2
```

## Ruby Version

```ruby
ruby 2.6.3p62 (2019-04-16 revision 67580) [x86_64-linux]
```

## Platform Version

ArchLinux

## Replication Case

```yaml
provisioner:
  name: chef_zero
  chef_license: accept 
```

## Kitchen Output

```ruby
>>>>>> ------Exception-------
>>>>>> Class: Kitchen::ActionFailed
>>>>>> Message: 1 actions failed.
>>>>>>     Failed to complete #converge action: [undefined method `name_from_mixlib' for #<LicenseAcceptance::Acceptor:0x00005617fec11d78>] on test-01-centos-7
>>>>>> ----------------------
```

**Comment by josharrington** [comment]
I had the same issue. Downgrading to 2.1.0 is my workaround for now

**Comment by tas50** [comment]
This has been resolved in the 2.2.3 release. Thanks for the report.
